content/yaml-quick-start/building-a-react-native-app.md

```diff
@@ -217,8 +217,7 @@ Add the following scripts just after the **Install npm dependencies**
 scripts:
   - name: Install Expo CLI and eject
```

Comment:
Perhaps we can change the script name as well because installing Expo CLI is not necessary at this point. Maybe something like Expo eject for Android and iOS